content/ecosystem/data.toml

```diff
@@ -1604,7 +1620,7 @@ categories = ["tools"]
 [[items]]
 name = "wgpu"
 source = "crates"
-categories = ["3drendering"]
+categories = ["2drendering", "3drendering"]
```

Comment:
I feel like this should be here too, if things like miniquad are in 2D rendering?

Reply:
:+1: Makes sense to me.